environment.yml

```diff
@@ -14,6 +14,18 @@ dependencies:
   - scipy>=1.0,<1.13
   - yamale>=4.0
   - h5py>=3.5
-  - s1reader>=0.2.0
+  - s1reader>=0.2.1
   - shapely>=1.8
   - requests>=2.0
+  - scikit-image
+# CI unit test
+  - pytest
+  - pluggy
+  - pytest-order
+# Dependencies for RAiDER
```

Comment:
Are we making `raider` a full requirement for COMPASS? 
we seem to have added a lot of it's dependencies now, and keeping this section in the main `environment.yml` file sort of implies we want Raider as a dependency... but we're not specifying `raider` or `raider-base`

Reply:
Based on what I experienced in beta release, installing RAiDER from conda-forge takes forever, and it adds way too much dependencies. So I am not a fan of adding `RAiDER` to `environments.yml`.